SQL/tgstation_schema.sql

```diff
@@ -254,6 +254,7 @@ CREATE TABLE `messages` (
   `round_id` int(11) unsigned NOT NULL,
   `secret` tinyint(1) unsigned NOT NULL,
   `expire_timestamp` datetime DEFAULT NULL,
+  `severity` tinyint(1) unsigned NULL,

```

Comment:
You should use an enum for this. An enum translates strings into a list of permitted non-mutable values represented by integers, which is essentially what you're doing here.

It also means you can do away with all your code that handles the severity changing between string and integer.

Reply:
I'm confident in that the code isn't going to need to build a list of permitted enums.
And while I also think it's quite unlikely the set would change, even if it did with the scale of our tables and infrequency of access isn't going to matter.

There are various other flaws to enums i.e. portability and cross-table use, but I don't believe they are a concern for us. Ours is a very simple use case.